scripts/zf_make_official_srpm

```diff
@@ -93,4 +96,9 @@ try gzip -v "${zf_tarball}"
 try mv -v "${zf_tarball}.gz" "${zf_tgz}"
 
 # Create SRPM.
-try rpmbuild -ts "${zf_tgz}"
+rpmbuild_args=()
+if [ -n "$outdir" ]; then
+  rpmbuild_args=(--define "_topdir $outdir");
```

Comment:
I was just aiming to get something that works for both jenkins and manual creation workflows. This was added so that the jenkins pipeline could pick up the packages relatively simply, I was having difficulties getting jenkins to find the packages using shell variables or absolute addresses so this was the solution I came up with. I think with a manual workflow it should be fine to ignore `--outdir` and configure where `rpmbuild` puts the packages.

Reply:
Agreed that `outdir` isn't the best name. I like you idea of the overridable array, when I come back this after the release I'll incorporate it.